ledger/src/leader_schedule.rs

```diff
@@ -48,6 +64,28 @@ impl LeaderSchedule {
     pub fn num_slots(&self) -> usize {
         self.slot_leaders.len()
     }
+
+    /// Returns indices starting from 'offset' where pubkey is the leader.
+    pub(crate) fn get_schedule(
```

Comment:
nit, how about `get_schedule -> get_leader_schedule_indices`

Reply:
Renamed it to `get_indices`. The object itself is called `leader_schedule`, so i think `leader_schedule.get_leader_schedule_indices` has unnecessary repetition.